README.md

```diff
@@ -45,87 +52,105 @@ module.exports = function(config){
+depends what framework that specific test runner supports. By default, this value is also used for `testFramework`.  
 
-#### Coverage analysis
-**Full notation:** `--coverageAnalysis perTest`    
-**Config file key:** `coverageAnalysis: 'perTest'`    
+#### Type of coverage analysis  
+**Full notation:** `--coverageAnalysis perTest`  
+**Config file key:** `coverageAnalysis: 'perTest'`  
 **Default value:** `perTest`  
+**Mandatory**: yes  
 **Description:**  
-The coverage analysis strategy you want to use.
-Stryker can analyse code coverage results. When doing so, it can potentially speed up mutation testing a lot, 
-because only the tests covering a particular mutation are tested for each mutant. 
-However this should *not* influence the resulting mutation testing score, only the performance.
-
-Possible values are: 
-* **off**: Stryker will not determine the covered code during the initial test run fase. All tests are always tested for each mutant during the mutation testing fase.
-* **all**: Stryker will determine the covered code of all tests during the initial test run fase. Only mutants that are actually covered by your test suite are tested during the mutation testing fase. This setting requires your test runner to be able to *report* the code coverage back to Stryker, which is the case for the stryker-mocha-runner and the stryker-karma-runner.
-* **perTest**: Stryker will determine the covered code per executed test during the initial test run fase. Only mutants that are actually covered by your test suite are tested during the mutation testing fase. Only those tests that cover a particular mutant are tested for each mutant. This requires your tests to be able to run independently of each other and in random order. In addition to the requirement for your test runner to be able to *report* the code coverage back to Stryker, your choosen testFramework also needs to support running code before and after each test and test filtering. 
-
-#### Reporter
+With `coverageAnalysis` you specify which coverage analysis strategy you want to use.  
+Stryker can analyse code coverage results. This can potentially speed up mutation testing a lot, as only the tests covering a 
+particular mutation are tested for each mutant. 
+This does *not* influence the resulting mutation testing score. It only improves performance.
+
+The possible values are: 
+* **off**: Stryker will not determine the code covered by tests during the initial test run phase. All tests will be executed for each mutant 
+during the mutation testing phase.
+
+* **all**: Stryker will determine the code covered by all tests during the initial test run phase. Only mutants actually covered by your 
+test suite are tested during the mutation testing phase. This setting requires your test runner to be able to report the code coverage back to Stryker. 
+Currently, only the `stryker-mocha-runner` and the `stryker-karma-runner` do this.  
```

Comment:
In fact, these are the only 2 runners available a.t.m. So you could also say all runners do this.